src/KawaBot.Tests/BasicControllerTests.fs

```diff
@@ -4,20 +4,36 @@ open System.Net
 open KawaBot.Feeder
 open KawaBot.Server
 open NUnit.Framework
+open Moq
+open System.Net.Http
+open System.Text
+open Newtonsoft.Json
 
 [<TestFixture>]
 [<Category(Categories.Integration)>]
 type BasicControllerTests =
     val private _server: Http.HttpListener
     val private _feeder: Feeder
+    val private _httpRequsterMock: Mock<IHTTPRequester>
```

Comment:
Lets use the Pascal case for abbreviations longer than 3 chars: `HTTP` -> `Http`.